roles/rke/tasks/registration.yaml

```diff
@@ -26,9 +26,9 @@
 - name: Install OpenStack Cloud Controller Manager
   shell: kubectl apply -f {{ item }}
   with_items:
-    - https://raw.githubusercontent.com/kubernetes/cloud-provider-openstack/master/cluster/addons/rbac/cloud-controller-manager-roles.yaml
-    - https://raw.githubusercontent.com/kubernetes/cloud-provider-openstack/master/cluster/addons/rbac/cloud-controller-manager-role-bindings.yaml
-    - https://raw.githubusercontent.com/kubernetes/cloud-provider-openstack/master/manifests/controller-manager/openstack-cloud-controller-manager-ds.yaml
+    - https://raw.githubusercontent.com/kubernetes/cloud-provider-openstack/master/manifests/controller-manager/cloud-controller-manager-roles.yaml
+    - https://raw.githubusercontent.com/kubernetes/cloud-provider-openstack/master/manifests/controller-manager/cloud-controller-manager-role-bindings.yaml
+    - https://gist.githubusercontent.com/almahmoud/b02942eb2b18ea14123e163e069a79e2/raw/b59139b17d697c9075ce3870ad7e88c567851d8a/openstack-ds.yaml
```

Comment:
Why is the custom gist needed? Can we check this in somewhere, maybe into the cloudman-boot repo itself?

Reply:
The control-plane vs master change for the daemonset node selector was the main issue. The custom gist was to make it work with 1.19 as before. With 1.24, the newer one from their repo works, but the patch under needs to change, and helm versions (notably for nginx) need to be upgraded. This was the easiest way to get it working in the meantime